src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -166,24 +173,19 @@ class AddressValidationModal extends React.Component {
     const {
       isAddressValidationModalVisible,
       addressValidationType,
-      suggestedAddresses,
       addressFromUser,
       validationKey,
       addressValidationError,
       closeModal,
       resetAddressValidation,
+      suggestedAddresses,
```

Comment:
No sense in just moving this, right?

Reply:
@erikphansen sometimes this will happen when I've made a change and then put it back without using revert.  I don't really see how this is a problem, though.  I just don't remember precisely where a prop was destructured.